pages/Announcements/Types/Reaction.md

```diff
@@ -0,0 +1,76 @@
+---
+name: "Type: Reaction"
+route: /Announcements/Types/Reaction
+menu: Announcements
+---
+
+# Reaction Announcement
+
+## Specification Status
+
+| Version | Status   |
+| ------  | -------- |
+| 1.0     | Proposed |
+
+## Details
+
+A Reaction Announcement is for publishing emoji reactions to anything with a [DSNP Announcement Id](/Identifiers#dsnp-announcement-id).
+
+## Fields
+
+| Field | Description | Serialization | Parquet Type | Bloom Filter |
+| ----- | ----------- | ------------- | ------------ | ------------ |
+| announcementType | Announcement Type Enum (`4`) | [hexadecimal](/Announcements/Overview#hexadecimal) | `INT32` | no |
+| createdAt | microseconds since Unix epoch | [hexadecimal](/Announcements/Overview#hexadecimal) | `INT64` | no
+| emoji | the encoded reaction | [UTF-8](https://datatracker.ietf.org/doc/html/rfc3629) | `BYTE_ARRAY` | YES
+| fromId | id of the user creating the relationship | [hexadecimal](/Announcements/Overview#hexadecimal) | `BYTE_ARRAY` | YES
+| inReplyTo | Target [DSNP Announcement Id](/Identifiers#dsnp-announcement-id) | [UTF-8](https://datatracker.ietf.org/doc/html/rfc3629) | `BYTE_ARRAY` | YES
+| signature | creator signature | [hexadecimal](/Announcements/Overview#hexadecimal) | `BYTE_ARRAY` | no
+
+## Field Requirements
+
+### announcementType
+
+- MUST be fixed to `4`
+
+### createdAt
+
+- MUST be set to the microseconds since Unix epoch at time of signing
+
+### emoji
+
+- Emoji fields must not be empty.
+- Emoji fields must only consist of [Unicode points](https://unicode.org/standard/standard.html) from `U+2000` to `U+2BFF`, from `U+E000` to `U+FFFF`, or from `U+1F000` to `U+10FFFF`.
+
+#### Examples
+
+All of the following should be considered valid emoji:
+
+```
+"😀", "🤌🏼", "👩🏻‍🎤", "🧑🏿‍🏫", "🏳️‍🌈", "🏳️‍⚧️", "⚛︎", "🃑", "♻︎"
+```
+
+None of the following should be considered valid:
+
+```
+"F", ":custom-emoji:", "<custom-emoji>", "ᚱ", "ᘐ", "״"
+```
+
+### fromId
+
+- MUST be a [DSNP User Id](/Identifiers#dsnp-user-id)
+- MUST be the [signer](/Announcements/Signatures) of the announcement
+
+### inReplyTo
+
+- MUST be a [DSNP Announcement Id](/Identifiers#dsnp-announcement-id)
+
+### signature
+
+- MUST be an [Announcement Signature](/Announcements/Signatures) over the all fields except the signature field.
+
+## Non-Normative
+
+### Likes
+
+Generic "likes" should default use the `"❤️"` or unicode `U+FE0F` as the emoji in the reaction.
```

Comment:
Why are we specifying this?

Reply:
This is in the non-normative section, so it is not part of the spec, but it is a suggestion.

I added it because I've already been asked the question by @pfrank13 actually.

I'm also ok with us removing it (in another PR) if we don't like it.